lib/Rex/Service/OpenBSD.pm

```diff
@@ -24,34 +24,17 @@ sub new {
   bless( $self, $proto );
 
   $self->{commands} = {
-    start   => '/etc/rc.d/%s start',
-    restart => '/etc/rc.d/%s restart',
-    stop    => '/etc/rc.d/%s stop',
-    reload  => '/etc/rc.d/%s reload',
-    status  => '/etc/rc.d/%s status',
-    action  => '/etc/rc.d/%s %s',
+    start        => '/usr/sbin/rcctl start %s',
+    restart      => '/usr/sbin/rcctl restart %s',
+    stop         => '/usr/sbin/rcctl stop %s',
+    reload       => '/usr/sbin/rcctl reload %s',
+    status       => '/usr/sbin/rcctl check %s',
```

Comment:
> I don't think that would be useful to map a check to rcctl check because it would break the compatibility of a Rexfile that we would use on another system.

Yeah, I meant keeping `status` as is (for backwards compatibility), but also adding `check` (to be inline with openbsd terms). If you think that wouldn't be that useful, it's OK like this (as we can add `check` later anytime when needed).

> What could be very useful is a flags => "-A"

I see, and agree :+1: If it's easy to add similar to `action`, then feel free to push one more commit to this branch. Otherwise I'd say let's keep this PR small and easy, and add `flags` separately.